config/jobs/kubernetes-sigs/cluster-api-provider-ibmcloud/cluster-api-provider-ibmcom-presubmits-release-0.4.yaml

```diff
@@ -0,0 +1,129 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-ibmcloud:
+  - name: pull-cluster-api-provider-ibmcloud-make-release-0-4
+    always_run: true
+    branches:
+      # The script this job runs is not in all branches.
+      - ^release-0.3
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-ibmcloud
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - command:
+        - "runner.sh"
+        - "./scripts/ci-make.sh"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230207-192d5afee3-1.25
+        imagePullPolicy: Always
+        resources:
+          requests:
+            memory: "6Gi"
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-ibmcloud
+      testgrid-tab-name: pr-make-release-0-4
+  - name: pull-cluster-api-provider-ibmcloud-test-release-0-4
+    always_run: true
+    branches:
+      # The script this job runs is not in all branches.
+      - ^release-0.4
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-ibmcloud
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230207-192d5afee3-1.25
+        imagePullPolicy: Always
+        command:
+        - "./scripts/ci-test.sh"
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-ibmcloud
+      testgrid-tab-name: pr-test-release-0-4
+  - name: pull-cluster-api-provider-ibmcloud-smoke-test-release-0-4
+    always_run: true
+    branches:
+      # The script this job runs is not in all branches.
+      - ^release-0-4
```

Comment:
@mkumatag, the jobs from here are pointing to `release-0-4` instead of `release-0.4` unlike the previous ones.

Reply:
copy paste error :P , fixed all of them, ptal